modules/Language.py

```diff
@@ -0,0 +1,139 @@
+import requests
+
+def get_Language(server, token, movie_id):
+    headers = {'X-Plex-Token': token, 'Accept': 'application/json'}
+    response = requests.get(f'{server}/library/metadata/{movie_id}', headers=headers)
+    data = response.json()
+
+    media = data['MediaContainer']['Metadata'][0]['Media'][0]['Part'][0]
+    if 'Stream' in media:
+        audio_streams = [stream for stream in media['Stream'] if stream['streamType'] == 2]
+
+        # Skip processing if there are multiple audio tracks
+        if len(audio_streams) > 1:
+            return None
+
+        # Mapping of common language codes to full names in alphabetical order
+        language_mapping = {

```

Comment:
Would it be possible to extend this to also support [language tags](https://www.rfc-editor.org/rfc/rfc9110.html#name-language-tags)?

Reply:
Like mi, en, da, etc? or what tag format?

Reply:
Also referred to as `locales`. Here's another [reference](https://en.wikipedia.org/wiki/Language_localisation). 

Personally example much of my library includes Arabic that's either dubbed into both classical/modern standard arabic (`ar`) or egyptian arabic (`ar-eg`).

Reply:
Another thing that will need to be updated along with this is the main [README](https://github.com/x1ao4/edition-manager-for-plex?tab=readme-ov-file#modules)

Reply:
> Also referred to as `locales`. Here's another [reference](https://en.wikipedia.org/wiki/Language_localisation).
> 
> Personally example much of my library includes Arabic that's either dubbed into both classical/modern standard arabic (`ar`) or egyptian arabic (`ar-eg`).

![image](https://github.com/x1ao4/edition-manager-for-plex/assets/28127566/776c83c2-9655-4ab1-b5ca-5a167b27ee5a)
My languages are based on language codes you would find in mkvtoolnix to define audio tracks

Reply:
I see. So would I be using [this designation](https://github.com/x1ao4/edition-manager-for-plex/pull/1/commits/ed376592824ea6d67901cf07780b492961403698#diff-fe786fec43e8e0969f38b72df1f80e9788f4865fdbc5d3b5ed5bbdf1691309c4R1886) going forward for any `ar-eg` files? 

<img src="https://github.com/x1ao4/edition-manager-for-plex/assets/326888/fa01784c-5805-42ab-8c4e-338a3e41332f" width="400" height="300"/>